client/scripts/views/pages/community_cards.tsx

```diff
@@ -78,62 +78,33 @@ class ChainCard implements m.ClassComponent<ChainCardAttrs> {
   }
 }
 
-type CommunityCardAttrs = { community: ChainInfo };
-
-class CommunityCard implements m.ClassComponent<CommunityCardAttrs> {
-  view(vnode) {
-    const { community } = vnode.attrs;
-
-    const redirectFunction = (e) => {
-      e.preventDefault();
-      localStorage['home-scrollY'] = window.scrollY;
-      m.route.set(`/${community.id}`);
-    };
+// Corresponds to 'ChainCategoryTypes' id numbering (which currently only includes DeFi and DAO)
+const filterMap = {
+  DeFi: 1,
+  DAO: 2,
+  ERC20: 3,
+  Cosmos: 4,
```

Comment:
the ERC20 should be pulled out into it's own array of approved network types for filtering (see `enum ChainNetwork`) of which we'll select a subset (currently, just ERC20).

Cosmos, Substrate, Ethereum, Near, & Solana can be pulled from the `enum ChainBase`. they should be in their own chainBase filter array.